Pipeline vehicle

LokeZhou opened the pull request:
*add ppvehicle_retrograde.md
*fix lane_seg preditor init

LokeZhou committed:
* add vehicle_pressing.py to judge vehicle press lane
* add lane_seg_infer.py to lane seg
* add pipeline vehicle retrograde:
* add vehicle_retrograde.py to judge vehicle retrograde
* add infer_cfg_vehicle_violation.yml to config vehicle press vehicle retrograde
* add lane_seg_config.yml to config lane seg

LokeZhou committed:
*add ppvehicle_press readme
*add ppvehicle_retrograde.md
*fix lane_seg preditor init

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_press.md:
````diff
@@ -0,0 +1,98 @@
+
+```bash
+# 预测单张图片文件
+python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_ppvehicle.yml \
````
这个命令中没有开启违法分析相关功能，可以用examples下的配置文件或者现在这个配置文件后面跟-o Module.enable=True打开功能

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_press.md:
````diff
@@ -0,0 +1,98 @@
+
+```bash
+#预测单个视频文件
+python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_ppvehicle.yml \
````
同上

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_press.md:
````diff
@@ -0,0 +1,98 @@
+    - 方法二：直接在命令行中增加`-o`，以覆盖配置文件中的默认模型路径：
+
+```bash
+python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_ppvehicle.yml \
````
同上

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_press.md:
```diff
@@ -0,0 +1,98 @@
+测试效果如下：
+
+<div width="1000" align="center">
+  <img src="../images/vehicle_press.gif"/>
```
图片是否缺失了

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_retrograde.md:
```diff
@@ -0,0 +1,101 @@
+[English](ppvehicle_press_en.md) | 简体中文
```
英文文档后面记得补充

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_retrograde.md:
````diff
@@ -0,0 +1,101 @@
+
+```bash
+#预测单个视频文件
+python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_ppvehicle.yml \
````
同上个文件中comment

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_retrograde.md:
```diff
@@ -0,0 +1,101 @@
+测试效果如下：
+
+<div width="1000" align="center">
+  <img src="../images/vehicle_retrograde.gif"/>
```
同上个文件comment

LokeZhou committed:
ppvehicle_press.md ppvehicle_retrograde.ma add gif image

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_retrograde.md:
```diff
@@ -0,0 +1,101 @@
+测试效果如下：
+
+<div width="1000" align="center">
+  <img src="https://raw.githubusercontent.com/LokeZhou/PaddleDetection/develop/deploy/pipeline/docs/images/vehicle_retrograde.gif"/>
```
这张图的逆行是自己翻转轨迹的结果吧，实际车辆并没有逆行，不能做为展示的demo。否则会被认为算法的问题

zhiboniu commented on deploy/pipeline/docs/tutorials/ppvehicle_press.md:
```diff
@@ -0,0 +1,98 @@
+测试效果如下：
+
+<div width="1000" align="center">
+  <img src="https://raw.githubusercontent.com/LokeZhou/PaddleDetection/develop/deploy/pipeline/docs/images/vehicle_press.gif"/>
```
图片上的车辆也没有压线的情况，作为案例不是很好。最好有个别压线的车辆样本展示效果。

LokeZhou committed:
*fix  ppvehicle press retrograde readme, test=document_fix
*add ppvehicle press retrograde en readme test=document_fix
*fix ppvehicle retrograde keep left bug

LokeZhou committed:
fix ppvehicle readme,test=document_fix

LokeZhou committed:
fix ppvehicle retrograde en readme,test=document_fix

jerrywgz commented on deploy/pipeline/docs/tutorials/ppvehicle_press.md:
```diff
@@ -0,0 +1,103 @@
+</div>
+
+## 方案说明
+车道线识别模型使用了[PaddleSeg](https://github.com/PaddlePaddle/PaddleSeg) 的超轻量分割方案。
```
可以加下压线的逻辑判断方案

LokeZhou replied:
已在最新commit添加

jerrywgz commented on deploy/pipeline/docs/tutorials/ppvehicle_retrograde.md:
```diff
@@ -0,0 +1,104 @@
+
+
+## 方案说明
+车道线识别模型使用了[PaddleSeg](https://github.com/PaddlePaddle/PaddleSeg) 的超轻量分割方案。
```
这里也可以加下逆行的判断方案说明

LokeZhou replied:
已在最新commit添加

LokeZhou committed:
ppvehicle_press ppvehicle_retrograde readme add sulution test=document_fix

LokeZhou committed:
fix ppvehicle press retrograde readme format test=document_fix
